### PR TITLE
page model

### PR DESCRIPTION
**Pull request by lk432**

### CODE REVIEW

File: kaeru/models.py [hunk 1]
```diff
@@ -27,3 +27,15 @@ class Code(models.Model):
...
 		return self.name
+
+class Page(models.Model):
+    page_name = models.CharField(max_length=200)   #Page name
+    page_create_date = models.DateTimeField('Date created') #creation date
+    page_modify_date = models.DateTimeField('Date modified') #modification date
+
+    project = models.ForeignKey(Project)
+    user = models.ForeignKey(User)
+    code = models.OneToOneField(Code)
```
Comment (fabianmuehlboeck) [inline kaeru/models.py]: I think according to the model in #13 this is a one-to-many field.

**Comment by fabianmuehlboeck** [comment]
Please remove the idea-files from the pull-request.

**Comment by lk432** [comment]
the idea files are removed

**Comment by Hapa29** [comment]
After I merged with this pull request, I'm getting this error now when I try to run the server. I don't think its a problem with my forked repo because I'm getting this error on my master branch, which I have kept exactly the same as the upstream repo.

error

**Comment by fabianmuehlboeck** [comment]
Sorry about that. I pushed an emergency fix (some tests are now going to fail, but the server should work). Sadly, getting the repo back to a state before the merge seems way too hard.

**Pull request merged**